worksystem_control_node:
  ros__parameters:
    longitudinal_velocity_kp: 1.0
    longitudinal_velocity_ki: 0.0
    longitudinal_velocity_kd: 0.5
    lateral_stanley_gain: 0.001
    lateral_stanley_softening_constant: 0.000001 # 1e-6
